app/templates/dev-flush-page.html

```diff
@@ -0,0 +1,118 @@
+{% set _theme="default" %}
+{% extends 'layouts/_base.html' %} {% block page_title %}EQ Development Flush Data{% endblock %} {% block content %}
+
+<!-- this page needs a total rethink, but this tidies things up a little for now -->
```

Comment:
is it worth doing this now? could it not use the dev-page just with a different button at the bottom?